lib/std/std.zig

```diff
@@ -146,6 +146,11 @@ pub const Options = struct {
     /// make a HTTPS connection.
     http_disable_tls: bool = false,
 
+    /// This enables `std.http.Client` to log ssl secrets to the file specified by the SSLKEYLOGFILE
+    /// env var.  Creating such a log file allows other programs with access to that file to decrypt
+    /// all `std.http.Client` traffic made by this program.
+    http_enable_ssl_key_log_file: bool = @import("builtin").mode == .Debug,
```

Comment:
Maybe this should default to false in all builds?

Reply:
Note that it also requires the env var to be set. So for the secrets to be logged the user (or parent process) has to explicitly ask for it. In both cases the secrets could have been accessed via other mechanisms (e.g. ptrace).